examples/OrderEdit/OrderEditSetStagedActionsAction.json

```diff
@@ -0,0 +1,6 @@
+{
+  "action": "setStagedActions",
+  "stagedActions": [
+    "recalculate"
```

Comment:
Still no valid action. Should be
```
"stagedActions": [
   {
      "action": "recalculate"
   }
]
```

But please check first if this action is stageable.

Reply:
switched to setCountry